gensrc/proto/cloud.proto

```diff
@@ -365,10 +366,21 @@ message RoutineLoadJobStatisticPB {
     optional int64 task_execution_time_ms = 5;
 }
 
+message StreamingTaskCommitAttachmentPB {
+    optional int64 job_id = 1;
+    optional int64 task_id = 2;
```

Comment:
what is task id? what is the different/relationship between task_id and job_id?

Reply:
The jobid is determined when creating the job, job will create task to load data, every task has its own task id.